samples/bookinfo/consul/bookinfo.yaml

```diff
@@ -18,15 +19,16 @@ services:
       - "9080"
 
   ratings-v1:
-    image: istio/examples-bookinfo-ratings-v1-envoy:0.2.4
+    image: istio/examples-bookinfo-ratings-v1-envoy:0.2.7
     networks:
       istiomesh:
     dns:
       - 172.28.0.1
       - 8.8.8.8
     dns_search:
         - service.consul
-    privileged: true
+    cap_add:
+      - NET_ADMIN
```

Comment:
fyi @GregHanson and @linsun - tested with cap net_Admin and it works. So no need to give elevated "priviledged:true".

Reply:
nice, thanks!!